nengo/config.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, configures):
         assert inspect.isclass(configures)
         self._configures = configures
         self._extraparams = {}
-        self._default_params = tuple(
+        self._defaultparams = tuple(
```

Comment:
Yeah, I'm fine either way; it's kind of lame right now actually that there's both `_extraparams` and `extra_params`, probably they shouldn't differ in the underscore.

Anyone have preferences for or against the underscore? I like them without, but it's not a strong preference.